styles/06-components/brand-theme.css

```diff
@@ -0,0 +1,89 @@
+ body {
+   color: var(--color-text);
+   background-color: var(--color-background);
+   min-height: 100vh;
+ }
+
+ .heading,
+ .text {
+   line-height: 1.75;
+   text-align: center;
```

Comment:
put it into some utility classes, that way you can have consistency in the future without a problem.

Reply:
This is a util, I missed these in the component library.  I wrote it separately so that I can easily add it to the file in the library.